srcpkgs/deheader/patches/0001-Fix-spelling-error-in-manpage.patch

```diff
@@ -0,0 +1,25 @@
+From e455d0a71612cdf614afaceaef10822454434d33 Mon Sep 17 00:00:00 2001
+From: Reiner Herrmann <reiner@reiner-h.de>
+Date: Sat, 23 Sep 2017 13:55:45 +0200
+Subject: [PATCH 1/3] Fix spelling error in manpage
```

Comment:
Not exactly mission critical bugfix :)

Reply:
:) yep, but still nice to fix some "issues"